README.md

```diff
@@ -16,5 +16,6 @@
 - [Ayúdame a encontrar mi cel](challenges/find-me.md)
 - [Unicorneando las PR](challenges/unicorn-pr.md)
 - [Bot Food](challenges/bot-food.md)
+- [eSport Analysis & Statistics App](challenges/panda-api.md)
```

Comment:
```suggestion
- [eSports Analysis & Statistics App](challenges/esports_app.md)
```

Reply:
We are not using underscore for the other files

Reply:
`- [eSports Analysis & Statistics App](challenges/esports-app.md)`